xrootd-ndn-oss-plugin/src/xrdndn-consumer.cc

```diff
@@ -276,8 +276,16 @@ void Consumer::onReadData(const ndn::Interest &interest,
         });
 }
 
+    
+
 ssize_t Consumer::Read(void *buff, off_t offset, size_t blen,
                        std::string path) {
+//This is potentially dangerous. You will flood the producer and packets will get dropped if you 
```

Comment:
For the moment we don't flood the producer because after each interest processEvents() on the interface is called and it will block until a response is received.  I agree, a congestion control will be better.

I can't merge this because is simply a comment in the source code. Please open an issue.

Reply:
Has been resolved.